MAUI/ImageEditor/text.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: Text Annotation in .NET MAUI ImageEditor control | Syncfusion
```

Comment:
title: Text Annotation in .NET MAUI Image Editor control | Syncfusion
description: Learn about the text annotation feature of Syncfusion .NET MAUI Image Editor(SfImageEditor) control.
platform: .NET MAUI
control: SfImageEditor
documentation: ug

Reply:
Changed a headings